apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LogTo.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Cloud.Logging.V2;
+using System;
+
+namespace Google.Cloud.Diagnostics.AspNetCore
+{
+    /// <summary>
+    /// The location log entries will be logged to.
+    /// </summary>
+    public enum LogToLocation
+    {
+        /// <summary>A Google Cloud Platform project.</summary>
+        Project,
+
+        /// <summary>A Google Cloud Platform organization.</summary>
+        Organization
+    }
+
+    /// <summary>
+    /// Options for log entries will be logged, such as a project or organization.
```

Comment:
"Options" has other connotations in these libraries. It might be better to change this to something like "Represents the location to which log entries will be sent, such as a project or organization."